### ISSUE TITLE
[makeotf] fail better when calling tx on UFO with missing lib.plist

### ISSUE THREAD

**Issue by frankrolf** [adobe-type-tools/afdko#1306]
We now have two issues (#1130 and #1305) which point at `tx` failing if the [optional](https://unifiedfontobject.org/versions/ufo3/lib.plist/) `lib.plist` file is missing in the UFO package. 

FontForge seems to be the only editor to write UFOs without `lib.plist`. Fonts with missing `lib.plist` file will also miss the `public.glyphOrder` key – which IMO is fairly essential.
I provided a Python workaround in #1130, but it would be nice for `tx`/`makeotf` to do that automatically, or (at least) provide a more helpful error message.

**Comment by josh-hadley** [comment]
Actually, looks like some modifications to `get_shell_command_output`:
https://github.com/adobe-type-tools/afdko/blob/c2a5bba6a8db1baea50cf3f3e946aba1e7438611/python/afdko/fdkutils.py#L126-L147
can improve this so the output becomes:
```
Command '['tx', '-dump', '-0', '/path/to/font.ufo']' returned non-zero exit status 3.
Failed to read lib.plist
tx: --- /path/to/font.ufo
tx: (ufr) source stream error
```
Basically we need to capture the `output` attribute of the `CalledProcessError` in that routine and print it.

Would that be better enough @frankrolf ?

**Comment by frankrolf** [comment]
That is definitely preferable to “non-zero exit status 6” :-)

**Comment by kaydeearts** [comment]
This was fixed in #1523. 
tx/makeotf should no longer error out if lib.plist is not present. 
Instead, you will see a warning message: `tx: (ufr) Warning: Unable to open lib.plist in source UFO font.`
as well as: `tx: (ufr) Warning: public.glyphOrder key is empty and does not contain glyph name for all x glyphs. Consider defining this in lib.plist.`
